<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>544</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>706</width>
    <height>544</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>706</width>
    <height>544</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MoreCoasts Editar Usuario</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>FormatFactoryAgregar un título.ico</normaloff>FormatFactoryAgregar un título.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(201, 230, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_NuevoCalculo">
      <property name="minimumSize">
       <size>
        <width>7</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
background-color: rgb(201, 230, 255);
}
QLabel{
background-color: #000000ff;
color::rgb(0, 74, 116);
border:8px solid#000000ff;
}
QLineButton{
border:0px;
color:rgb(255, 255, 255);
border-bottom:2px solid rgb(61,61,61);
}
QPushButton{
background-color:rgb(0, 74, 116);
border-radius:15px;
color: rgb(255, 255, 255);
}

QLineEdit{
background-color: rgb(255, 255, 255);
}
QPushButton:hover{
background-color: rgb(0, 12, 255);
border-radius:15px;
color: rgb(0, 0, 0);
}

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>51</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>Agregar_un_título-removebg-preview.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_SGCTS_GeoCuba">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>51</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Cooper Black</family>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #000000ff;
color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;MoreCoasts&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_NuevoCalculo">
         <property name="minimumSize">
          <size>
           <width>641</width>
           <height>61</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>61</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 74, 116);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Editar Usuario&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>8</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(143, 170, 163);</string>
         </property>
         <property name="text">
          <string>Los campos con (*)  son obligatorios</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_DensidadArena">
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Nombre*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Nombre">
           <property name="minimumSize">
            <size>
             <width>121</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>121</width>
             <height>41</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>IBeamCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="maxLength">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_IndiceR">
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Correo*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Correo">
           <property name="minimumSize">
            <size>
             <width>121</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>121</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="maxLength">
            <number>50</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_DensidadMar">
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Primer Apellido*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_PrimerApellido">
           <property name="minimumSize">
            <size>
             <width>121</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>121</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="maxLength">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Ubicacion">
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Nombre de Usuario*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_NombreUsuario">
           <property name="minimumSize">
            <size>
             <width>121</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>121</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="maxLength">
            <number>20</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_CoeficientePorocidad">
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Segundo Apellido*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_SegundoApellido">
           <property name="minimumSize">
            <size>
             <width>121</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>121</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="maxLength">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Ubicacion_2">
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Contraseña*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Contrasena">
           <property name="minimumSize">
            <size>
             <width>121</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>121</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="maxLength">
            <number>25</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_AnguloR">
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Tipo de Trabajador*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_TipoTrabajador">
           <property name="minimumSize">
            <size>
             <width>121</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>121</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="maxLength">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Ubicacion_3">
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Confirmar Contraseña*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_ConfirmarConstrasena">
           <property name="minimumSize">
            <size>
             <width>121</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>121</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="maxLength">
            <number>25</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_Error">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 1, 5);</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Label Error&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="but_actualizarUsuario">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>91</horstretch>
             <verstretch>31</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>91</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: rgb(11, 15, 255);
color: rgb(0, 0, 0);
border-radius:8px;
}

QPushButton:hover{
background-color: rgb(16, 132, 255);
border-top-left-radius:12px;
border-bottom-left-radius:12px;

}</string>
           </property>
           <property name="text">
            <string>Actualizar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="butCancelarEditarUsuario">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>91</horstretch>
             <verstretch>31</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>91</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
color: rgb(255, 255, 255);
	background-color: rgb(221, 1, 5);
border-radius:8px;
}

QPushButton:hover{
background-color: rgb(255, 120, 129);
border-top-left-radius:12px;
border-bottom-left-radius:12px;

}</string>
           </property>
           <property name="text">
            <string>Cancelar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
